02-aws-lambda-deployment/src/lambda-main.ts

```diff
@@ -0,0 +1,46 @@
+import { APIGatewayProxyHandler, Context } from 'aws-lambda'
+import { NestFactory } from '@nestjs/core'
+import { AppModule } from './app.module'
+import { Server } from 'http'
+import { ExpressAdapter } from '@nestjs/platform-express'
+import { createServer, proxy } from 'aws-serverless-express'
+import { eventContext } from 'aws-serverless-express/middleware'
+import * as express from 'express'
+import { Logger } from 'nestjs-pino';
+import { ValidationPipe } from '@nestjs/common';
+
+import { AppLogger } from './shared/logger/logger.service'
+import { RequestIdMiddleware } from './shared/middlewares/request-id/request-id.middleware';
+
+let cachedServer: Server
+
+const bootstrapServer = async (): Promise<Server> => {
+  const expressApp = express()
+  const adapter = new ExpressAdapter(expressApp)
+
+  const app = await NestFactory.create(AppModule, adapter, {
+    logger: false,
+  })
+
+  app.use(eventContext())
```

Comment:
@yashmurty 
Write comments here showing which lines need to be removed in comparison with `main.ts` file.
This will help people migrating from newer versions of starter-kit to this `AWS Lambda` deployment.